tests/conftest.py

```
@@ -78,3 +80,68 @@ def _override_get_db():
+        token = response.json()["access_token"]
+
+    client.headers["Authorization"] = f"Bearer {token}"
+    return client
+
+
+@pytest.fixture
+def storage_location(db_session: Session):
+    """Fixture for a ColdStorageLocation object."""
+    location = ColdStorageLocation(
+        name="Test Cold Storage",
+        path="/tmp/cold_storage",
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/cold_storage"

(S108)